content/guides/virtualizing/vmware-fusion.md

```diff
@@ -0,0 +1,104 @@
+There are several methods offered to install Haiku. Among those, only the vmdk and the iso methods are suitable for running Haiku easily on VMware Fusion. Basically these are the two methods that will be shown in this guide. The easiest of all is utilising the vmdk which is already properly configured.
+
+> Running Haiku from a VM image is recommended to avoid any possible problems. Unless it does not work, or you would like to simulate an actual installation, do stick to this method.
+
+The required files can be found at on the Get-Haiku page of this website which is located [here](http://www.haiku-os.org/get-haiku).
+
+
+Both the vmdk and iso are available there, do select the closest mirror to enjoy higher transfer rates. Verify using the checksums to make sure that the downloaded files are not corrupted as they are big files. VMware Fusion is available for free on their website which can be found [here](http://www.vmware.com/products/fusion/).
+
+##### Go to section:
+
+*   [Running Haiku from a VM image.](#part_vmimage)
+*   [Installing and running Haiku from an ISO image.](#part_iso)
+*   [Additional Steps.](#part_additional)
+*   [Troubleshooting.](#part_trouble)
+
+### Running Haiku from a VM image.
+
+##### Step 1\. Downloading Haiku.<a name="part_vmimage"></a>
+
+After downloading the VMDK image, unzip the file and in the folder, you will see a .vmx file.
+
+##### Step 2\. Opening the file.
+
+Open the .vmx file and open with VMware Fusion. VMware Fusion will prompt if you would like to upgrade this Virtual Machine, it is totally up to you if you want to as an ungraded version offers support for new features, however, cannot be used with older versions of VMware Fusion. It is possible if you want to downgrade the Virtual Machine. After selecting your choice, VMware Fusion will start to boot Haiku.
+
+##### Step 3\. Finishing Up.
+
+Wait for Haiku to boot and Haiku is now up and running! It is convenient, time-saving and easier to manage.  
+
+### Installing and running Haiku from an ISO image. <a name="part_iso"></a>
+
+This method is pretty much similar to installing on an actual machine, but a little more tedious than running Haiku from a VM image
+
+##### Step 1\. Downloading Haiku.
+
+After downloading the ISO image, unzip the file and in the folder, you will see a .iso file. Under Virtual Machine Library, which you can use the shortcut <kbd>Shift</kbd>+<kbd>cmd</kbd>+<kbd>L</kbd>, click Add and then New.
+
+##### Step 2\. Creating a virtual machine.
+
+![](https://raw.githubusercontent.com/haiku/website/master/static/files/new-vm-step-1.png)
+
+![](https://raw.githubusercontent.com/haiku/website/master/static/files/new-vm-step-2.png)
+ 
+![](https://raw.githubusercontent.com/haiku/website/master/static/files/new-vm-step-3.png)
```

Comment:
These URLs need to match the other ones (see one of the merged files for this), they can't point to GitHub. Also, you can delete the added images, these links point to the existing ones.